suave/launch/sauve.launch.py

```diff
@@ -67,32 +54,42 @@ def generate_launch_description():
         }],
     )
 
-    pipeline_metacontrol_node = Node(
-        package='pipeline_inspection_metacontrol',
-        executable='pipeline_metacontrol_node',
+    pipeline_detection_wv_node = Node(
+        package='suave',
+        executable='pipeline_detection_wv',
         output='screen'
     )
 
-    spiral_lc_node = Node(
-        package='pipeline_inspection_metacontrol',
-        executable='spiral_lc_node',
+    spiral_search_node = Node(
+        package='suave',
+        executable='spiral_search',
         output='screen'
     )
 
-    follow_pipeline_lc = Node(
-        package='pipeline_inspection_metacontrol',
-        executable='follow_pipeline_lc',
+    follow_pipeline_node = Node(
+        package='suave_metacontrol',
```

Comment:
should line 70 still say suave_metacontrol?

Reply:
No, it is wrong. Thanks for pointing out.